src/openrct2/sprites.h

```diff
@@ -746,6 +751,20 @@ enum {
 	SPR_G2_SORT = SPR_G2_BEGIN + 77,
 	SPR_G2_COPY = SPR_G2_BEGIN + 78,
 	SPR_G2_PASTE = SPR_G2_BEGIN + 79,
+
+	SPR_CSG_BEGIN = 393216,
```

Comment:
Well, apparently it does need a comment, as it isn't obvious to Marijn or to me.